### ISSUE TITLE
Can't use server path (eg. \\NAS) in config.ini as rootdirectory

### ISSUE THREAD

**Issue by Dexyne** [Nandaka/PixivUtil2#471]
Hello,

### Prerequisites

* [x] Did you read FAQ section?
* [x] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

Can't use server path (like a NAS) as a _rootdirectory_ in config.ini.

### Steps to Reproduce

1. Write a path like \\\My-Nas\pixiv as _rootdirectory_ in config.ini
2. Download images (executable is on Windows on C: drive)
3. PixivUtils puts the files into C:\My-Nas\pixiv folder instead, while showing the path as \My-Nas\pixiv\path\to\pixiv\image when downloading

I also tried to put the executable on the NAS and executing both from Windows and the NAS through SSH. From Windows it does the same as if I execute it from C: while with SSH it does not works as it uses Linux (I suppose some other build or tweaking would be needed but I don't have a Linux installed yet apart from the NAS).

**Expected behavior:** Put the files into the right folder, eg. \\\My-Nas\pixiv.

**Actual behavior:** Files go to C: instead of server (NAS) folder.

### Versions

PixivUtil20190212

**Comment by Nandaka** [comment]
map those network path as network drive (e.g. z: drive) or add it as network location (map to local folder)

**Comment by Dexyne** [comment]
Thanks for the answer.

If I map as network drive, am I forced to direct the path to the target folder? (I tried using the main folder first (home) but couldn't move from it and used the 'pixiv' folder directly, so Z: pointing to \\NAS\pixiv) 
If so, it's not the best solution, as it would mean to create a drive letter for every folder if needed or removing it and make it again. Well, not like I need to use 10 letters as of now...

As for the network location, I think I already have it like that (if you mean using \\My-NAS like a folder through Windows Explorer).

Otherwise the network drive solution works.
